examples/discord-bots/memory-price-bot.js

```diff
@@ -0,0 +1,19 @@
+const axios = require("axios");
+const Limestone = require("../../index.js");
+const { runSimpleDiscordBot } = require("./simple-discord-bot");
+
+async function getCurrentARPricePerGB() {
+  const response = await axios.get("https://arweave.net/price/1048576");
```

Comment:
Hmm, what is "1048576", there should be data size in bytes.

Reply:
I will change it to 1073741824